content/millennium/r4/clinical/care-provision/care-plan.md

```diff
@@ -83,9 +83,9 @@ Notes:
   - Only supports the code `assess-plan`.
   - This can be combined with the patient parameter.
 
-- The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-
-- The `_revinclude` parameter may be provided with the `_id/patient` parameter. Example: `_id=178866310&_revinclude=Provenance:target`
+- The `_revinclude` parameter 
+  - May be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+  - May be provided with the `_id/patient` parameter. Example: `_id=178866310&_revinclude=Provenance:target`
```

Comment:
let's split _id and patient so it doesn't look like a single parameter

Reply:
Adjusted in latest commit to the branch
![image](https://github.com/cerner/fhir.cerner.com/assets/144141792/14c6a10b-dcb5-4fb1-b483-5dfe6b4b7ca3)